hack/kiali/install-ossm-release.sh

```
@@ -0,0 +1,267 @@
+elif [ "${_CMD}" == "status" ]; then
+
+  status_servicemesh_operators
+  status_kiali_operator
+  status_istio
+  status_kiali_cr
+  status_ossmconsole_cr
+
+elif [ "${_CMD}" == "kiali-ui" ]; then
+
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+    kiali_url="http://$(${OC} -n ${CONTROL_PLANE_NAMESPACE} get route kiali -o jsonpath='{.spec.host}' 2> /dev/null)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)